types/trace-events.d.ts

```diff
@@ -0,0 +1,590 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// https://github.com/Hoten/chrome-trace-events-tsc/blob/master/dist/lh-trace-events.d.ts
+
+export namespace _TraceEvent {
+  interface Base {
```

Comment:
`EventBase`? I'm kind of divided on this cause vscode will show the full qualified type `TraceEvent.Base`, but on the other hand, reading the file it's easy to forget what that's referring to.

Maybe another, more descriptive word? Something about common properties or...

Reply:
I had to make all the type references use absolute paths (`TraceEvent.Base` instead of `TraceEvent`). It made the `print` method much simpler in the generation code. So now you see `extends TraceEvent.Base` all throughout this file.